lighthouse-core/lib/page-functions.js

```diff
@@ -218,6 +218,52 @@ function getNodeSelector(node) {
   return parts.join(' > ');
 }
 
+/**
```

Comment:
can you add an explanatory comment for the function? From just the name I would assume it's just checking if the element itself is `position: fixed`, so a sentence explaining why you might need to check for more would be 👍